src/app/GalleryManager.jsx

```diff
@@ -84,17 +84,18 @@ export default class GalleryManager extends Component {
     return (
       <div>
         <div style={styles.galleryTitle}>
-          <TextField
-            value={gallery.headline}
-            onChange={this.setHeadline.bind(this)}
+          <input
             style={styles.galleryTitles}
-            label="Write a headline (optional)" />
+            type="text"
+            value={gallery.headline}
+            placeholder="Headline"
+            onChange={this.setHeadline.bind(this)} />
```

Comment:
Could we bind this in the constructor?

this.setHeadline = this.setHeadline.bind(this)  // In the constructor

And onChange={this.setHeadline} // In the render function